sunspot/Gemfile

```diff
@@ -3,6 +3,7 @@ source "http://rubygems.org"
 gem 'sunspot_solr', :path => File.expand_path('../../sunspot_solr', __FILE__)
 
 gemspec
+gem "rsolr", git: "ssh://git@phabricator.docurated.rocks:2222/diffusion/RSOLR/rsolr.git", branch: "no-deprecation"
```

Comment:
Not going to work, when we bundle install our gems there is no SSH access to phabricator.